src/pages/getMoviesByCategory.js

```diff
@@ -0,0 +1,11 @@
+import { movieListView } from "../views/movieListView.js";
+export const getMoviesByCategory = (event) => {
+  const categoryId = event.target.id;
+
+  const movies = JSON.parse(localStorage.getItem("moviesList"));
+  const filteredMovies = movies.filter((movie) => {
+    return movie.genre_ids.includes(Number(categoryId));
+  });
```

Comment:
Here you are using `.filter()` correctly. But you can use shorthand notation here:

```suggestion
  const filteredMovies = movies.filter((movie) => movie.genre_ids.includes(Number(categoryId)));
```